author-guide.qmd

```diff
@@ -85,6 +85,16 @@ An example authorship statement might read:
 
 > **Aggie Eilwen**: Investigation, Data Curation, Writing -- Original Draft; **Pedro Ellar**: Writing -- Review & Editing; **Alena Ennio**: Writing -- Review & Editing, Supervision, Funding Acquisition.
 
+_Note:- Generative AI tools and technologies, such as ChatGPT, may not be listed as authors of an article published at JoVI. The use of generative AI tools and technologies to create content is permitted but must be fully disclosed in the article. For example, the authors could include the following statement in the Acknowledgements section of the article: 
```

Comment:
I'd suggest making this statement as a separate section to make the job of AE and reviewer easier. Perhaps we should add this into the templates too.

Reply:
(reminder:) If you change this to a separate section, there's a text below that also refer to the "acknowledgement section."

Reply:
OTOH, we could just request to integrate any such statement it into the authorship section, then we have it all in one place.

Reply:
(instead of into acknowledgements, which should be optional)